src/databricks/labs/remorph/helpers/reconcile_config_utils.py

```diff
@@ -0,0 +1,390 @@
+import json
+import logging
+import webbrowser
+
+from databricks.connect import DatabricksSession
+from databricks.labs.blueprint.installation import Installation, SerdeError
+from databricks.labs.blueprint.tui import Prompts
+from databricks.labs.remorph.config import TableRecon, ReconcileConfig, get_dialect
+from databricks.labs.remorph.reconcile.connectors.source_adapter import create_adapter
+from databricks.labs.remorph.reconcile.constants import SourceType
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.platform import ResourceDoesNotExist, NotFound, PermissionDenied
+
+logger = logging.getLogger(__name__)
+
+README_RECON_CONFIG_NOTEBOOK = """# Databricks notebook source
+# MAGIC %md
+# MAGIC # Recon Config setup instructions (see [README](readme_link))
+# MAGIC
+# MAGIC Production runs are supposed to be triggered through the following jobs: job_links
+# MAGIC
+# MAGIC **This notebook is overwritten with each `remorph` generate-recon-config.**
+
+# COMMAND ----------
+
+# MAGIC %pip install remorph_VERSION-py3-none-any.whl
```

Comment:
upload wheel to WSFS and replace the link in here - see how to do it in https://github.com/databrickslabs/blueprint?tab=readme-ov-file#publishing-wheels-to-databricks-workspace

Reply:
template added just for reference, we'll add more details in next PR